wfl/Profiles/FoodOrderCollectionProfile.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Hypermedia.Affordances;
+using Hypermedia.Transforms;
+using Hypermedia;
+using Symphono.Wfl.Models;
+using Symphono.Wfl.Controllers;
+using System;
+
+namespace Symphono.Wfl.Profiles
+{
+    public class FoodOrderCollectionProfile : EntityProfile<FoodOrderCollection>
+    {
+        protected override void Configure(EntityConfiguration<FoodOrderCollection> configuration)
+        {
+            configuration
+                .UseRepresentationTransform(representation => representation
+                    .WithRepresentation("food-order-collection")
+                    .WithRepresentation("collection")
+                )
+                .UseEnumerableEmbeddedSubEntityTransform(
+                    c => c.FoodOrders,
+                    embedded => embedded
+                        .WithRelation("item")
+                        .WithConfiguration(embeddedConfiguration => embeddedConfiguration
+                            .UseProfile<FoodOrder, FoodOrderProfile>()
+                        )
+                )
+                .WithLink<FoodOrderCollection, FoodOrdersController>(
+                    "self",
+                    c => fc => fc.GetAsync(c.Criteria)
+                )
+                .UseActionTransform(actions => actions
```

Comment:
To answer your question about criteria-applier, I do intend to make those changes but haven't gotten to it yet. In regards to providing a link to show what values are allowed for enumeration, I'll add that.